### ISSUE TITLE
Visual Studio does not highlight available coverage information when opening covered files

### ISSUE THREAD

**Issue by klaus1212** [FortuneN/FineCodeCoverage#447]
### Installed product versions
- Microsoft Visual Studio Professional 2022 17.10.4 (C++)
- This extension: 1.1.223

### Description
Fine Code Coverage view shows line coverage of my C++ files.
When I click a file in the coverage report, visual studio opens the file but it does not highlight the lines covered/ not covered in any way.

### Steps to recreate
1. Install the Fine Code Coverage extension
2. Apply Ms code coverage
3. Run unittests
4. Open Fine Code Coverage report
5. Click on a file with some coverage
6. None of the lines in the file are highlighted ![Coverage not highlighted](https://github.com/user-attachments/assets/4c77db1e-ba4d-4a7e-9949-c1e12d658063)

### Current behavior
Files shows line coverage information in Fine Code Coverage report, but none of the files highlights coverage information when opening a file.

### Expected behavior
Opening a file should highlight the coverage information available in the file.

**Comment by tonyhallett** [comment]
Have you set ShowEditorCoverage as true ?

<img width="955" alt="image" src="https://github.com/user-attachments/assets/5969ae30-d7a3-4788-9656-4ea00c22ee3b">

and 

<img width="302" alt="image" src="https://github.com/user-attachments/assets/f54939c7-3759-4cec-a30b-aa9f1d94d950">

**Comment by klaus1212** [comment]
Yes I have, these are my settings

![image](https://github.com/user-attachments/assets/9ba8712a-ccd5-45f6-a98b-c10c4805e2be)

**Comment by tonyhallett** [comment]
Can you share your solution or provide a minimal solution that exhibits this behaviour. What testing framework are you using ?

**Comment by tonyhallett** [comment]
Please do and I will have a look.

Just note that FCC was developed for C#.  The availability of microsoft code coverage and its ability to provide coverage for C++ allowed for the disclaimer 

> Supports C++ ! Note that FCC has not been properly tested with C++ projects but with a simple C++ class, tested with Google Test.

**Comment by klaus1212** [comment]
Here is the cobertura file, I renamed it to txt since xml is not supported.
[Cobertura.txt](https://github.com/user-attachments/files/16457378/Cobertura.txt)

**Comment by tonyhallett** [comment]
Is there editor marks for the cpp files ?

**Comment by klaus1212** [comment]
Hi,

While I was creating the example project, I earlier in this thread agreed to return with, I found out that my problem was my custom fcc-ms-runsettings-template.xml file.
Intially the example project I was working on posting here did not contain the ms-runsettings-template.xml file.
Hence I noticed that coverage worked with source file highlighting and everything :-)

Digging into it I noticed I was missing a ModulePath include for my unittest executables in the fcc-ms-runsettings-template.xml.
Therefore when I updated fcc-ms-runsettings-template.xml to use the FCC autogenerated modulepath (%fcc_modulepaths_include%) it worked :-)

So to conclude I have line coverage working - I get a report and source files highlighted with coverage.
Only thing I do not see is branch coverage? I only see line coverage.

Is branch coverage not part of FCC when using MS Code Coverage for FCC
